ui/app/pages/create-account/connect-hardware/index.js

```diff
@@ -129,7 +125,7 @@ class ConnectHardwareForm extends Component {
         }
       })
       .catch((e) => {
-        const errorMessage = e.message
+        const errorMessage = e.message || e
```

Comment:
Ah - so a _string_ is being thrown in this case? That explains it.

It might be helpful to more explicitly check the type here. A proper `Error` instance is _supposed_ to always have a `message`, but it does default to an empty string, which would then result in `errorMessage` being set to an object, so this'd still blow up.

Something like this might work:
```suggestion
        const errorMessage = typeof e === 'string' ? e : e.message
```